single_subject/batch_single_subject.sh

```diff
@@ -146,6 +146,37 @@ sct_compute_compression -i t2_compressed_seg.nii.gz -vertfile t2_compressed_seg_
 
 
 
+# Registering lumbar data to the PAM50 template
+# ======================================================================================================================
+cd ../t2_lumbar
+
+# Crop full-body image to isolate the lumbar region (lowest 200 axial slices)
+sct_crop_image -i t2.nii.gz -zmax 200
+
+# Use lumbar-specific `sct_deepseg` model to segment the spinal cord
+sct_deepseg -i t2_crop.nii.gz -task seg_lumbar_sc_t2w
+
+
+# Generate labels for the 2 spinal cord landmarks: cauda equinea ('99') and T9-T10 disc ('17')
+# Note: Normally this would be done manually using fsleyes' "Edit mode -> Create mask" functionality. (Uncomment below)
+#
+# fsleyes t2.nii.gz &
+#
+# However, since this is an automated script with example data, we will place the labels at known locations for the
+# sake of reproducing the results in the tutorial.
+sct_label_utils -i t2_crop.nii.gz -create 27,80,80,60 -o t2_crop_label.nii.gz
+sct_label_utils -i t2_crop_label.nii.gz -create-add 27,76,187,17 -o t2_crop_label.nii.gz
+
+# Register the image to the template using segmentation and labels
+# TODO: Uncomment when https://github.com/spinalcordtoolbox/PAM50/pull/27 gets merged and a new PAM50 release is made.
+#sct_register_to_template -i t2_crop.nii.gz \
+#                         -s t2_crop_seg.nii.gz \
+#                         -ldisc t2_crop_label.nii.gz \
+#                         -c t2 -qc qc \
+#                         -param step=1,type=seg,algo=centermassrot:step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,slicewise=0:step=3,type=im,algo=syn,metric=CC,iter=3,slicewise=0
```

Comment:
Note: The actual registration step causes an error, because the PAM50 release used in SCT's master branch does not yet have the cauda equinea label (`60`), see: 

- https://github.com/spinalcordtoolbox/PAM50/pull/27